lib/chippy_web/live/sprint_live/show.ex

```diff
@@ -80,17 +93,20 @@ defmodule ChippyWeb.SprintLive.Show do
 
   def update_sprint(sprint_id, new_sprint, socket) do
     PubSub.broadcast(Chippy.PubSub, "sprint:" <> sprint_id, {:sprints, :update, new_sprint})
-    {:noreply, assign(socket, sprint: new_sprint)}
+    {:noreply, assign(socket, %{sprint: new_sprint, online_users: online_users(sprint_id)})}
   end
 
-  def handle_info({:sprints, :update, new_sprint}, socket) do
-    {:noreply, assign(socket, sprint: new_sprint)}
+  def handle_info(
+        {:sprints, :update, new_sprint},
+        %{assigns: %{sprint_id: sprint_id}} = socket
```

Comment:
Lost me here as well :) I don't understand how a match operator works when in a params section. I *think* it's somehow destructuring the socket to grab the `sprint_id` from it, while also preserving the `socket` so that both variables can be used in the function body.

Reply:
That is exactly what it's doing (as far as I understand, at least)! I'm following @nmashton's lead here.